core-api/src/main/java/com/sequenceiq/cloudbreak/doc/ModelDescriptions.java

```diff
@@ -277,6 +277,11 @@ private ModelDescriptions() {
         public static final String AZURE_LB_NAME = "The Azure Load Balancer name";
         public static final String AZURE_LB_AVAILABILITY_SET = "The availability set that contains the instances recieving traffic from the load balancer " +
                 "on the specificed port.";
+        public static final String LOAD_BALANCER_GCP = "The GCP load balancer information";
+        public static final String TARGET_GROUP_GCP = "The GCP instance group information";
+        public static final String GCP_LOAD_BALANCER_NAME = "The name in GCP of the load balancer forwarding rule";
+        public static final String GCP_LB_INSTANCE_GROUP = "The instance group that with instances reciving traffic from the loadbalancer for this port";
```

Comment:
Nit: Could you reword this a little? Just not sure what exactly you mean by this. Also "receiving" was misspelled.